### PR TITLE
[cloud images] rename `build_id` to `creation_timestamp` tag

### PR DESCRIPTION
**Pull request by yaronkaikov**
As part of my work in https://github.com/scylladb/scylla-pkg/issues/3827, i need to delete old Azure images based on creation date, it seems that Azure doesn't have this information (make no sense), to overcome this we should have a creation_timestamp date tag so we can use.

Also noticed that we don't really use the `build_id` tag, so replacing it for all cloud images

### CODE REVIEW

**Comment by yaronkaikov** [comment]
Verified with https://jenkins.scylladb.com/job/scylla-master/job/releng-testing/job/next-machine-image/312/

**Comment by fruch** [comment]
`build_id` is used by SCT, but I think we can drop it, or also replace it with the timestamp like in this change

**Review by benipeled — DISMISSED** [review]

File: packer/build_image.sh [hunk 1]
```diff
@@ -76,8 +76,8 @@ while [ $# -gt 0 ]; do
             shift 2
             ;;
         "--build-id")
```
[the remark this answers is not in the document]
Comment (yaronkaikov) [reply packer/build_image.sh]: we can completely remove it as we now use date as default

**Review by Annamikhlin — APPROVED** [review]

**Comment by yaronkaikov** [comment]
Waiting for scylladb/scylla-cluster-tests#7397

**Comment by yaronkaikov** [comment]
> Waiting for [scylladb/scylla-cluster-tests#7397](https://github.com/scylladb/scylla-cluster-tests/issues/7397)

This was merged. Verified again with https://jenkins.scylladb.com/job/scylla-master/job/releng-testing/job/next-machine-image/319/

**Pull request merged**